core/src/main/java/org/elasticsearch/action/index/IndexRequest.java

```diff
@@ -49,14 +49,14 @@
 /**
  * Index request to index a typed JSON document into a specific index and make it searchable. Best
  * created using {@link org.elasticsearch.client.Requests#indexRequest(String)}.
- * <p>
```

Comment:
I just had enough fighting the linter when intelij added a <p/> and the linter got annoyed. Bugs me to have a unclosed <p> (although I believe it's strictly speaking ok).